hooks/pre_gen_project.py

```diff
@@ -0,0 +1,6 @@
+{{ cookiecutter.update({"package_name": "inmanta-module-"+cookiecutter.module_name.lower().replace("_","-") }) }}
```

Comment:
Could you add a description at the top of the file explaining the purpose of this script and the semantics (what they represent as well as what naming conventions they follow) of both variables it introduces?